.claude/commands/jira-task.md

```
@@ -0,0 +1,173 @@
+
+### API Token 발급
+
+1. https://id.atlassian.com/manage/api-tokens
+2. "Create API token" 클릭
+3. 토큰 복사 → 환경 변수에 설정
+
+---
+
+## 에러 처리
+
+### Issue 없음
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)